build/index.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
+// eslint-disable-next-line no-console
+console.log('Hello world!');
```

Comment:
Why do you have this build folder? It does not look like React distribution? Maybe it was inherited from the template and needs to be removed?